Rakefile

```diff
@@ -699,15 +699,6 @@ def generate_kokoro_configs
           config = ERB.new(File.read("./.kokoro/templates/#{os_version}.cfg.erb"))
           f.write(config.result(binding))
         end
-
```

Comment:
Why was this removed?

Reply:
It should've been removed earlier. For kokoro tasks there is a github side and a google3 side. In my ideal world you could be able to generate configs for both from this repo. I'm not at all sure that that is allowed. The code I deleted is an incomplete, leftover fragment from that exploration.

Reply:
kk